wallet/chain/p/builder/builder.go

```diff
@@ -1813,12 +1879,12 @@ func (b *builder) spend(
 	return s.inputs, s.changeOutputs, s.stakeOutputs, nil
 }
 
-func (b *builder) authorizeSubnet(subnetID ids.ID, options *common.Options) (*secp256k1fx.Input, error) {
-	ownerIntf, err := b.backend.GetSubnetOwner(options.Context(), subnetID)
+func (b *builder) authorize(ownerID ids.ID, options *common.Options) (*secp256k1fx.Input, error) {
```

Comment:
why we renamed it to ownerID?

Reply:
because it can either be a `subnetID` or a `validationID`

Reply:
hmm seems we have commented it 👍